Tools/gsc/finalize_manifests.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import subprocess
+import re
+import string
+import argparse
+
+def is_ascii(chars):
+    return all(ord(c) < 128 for c in chars)
+
+def generate_trusted_files(root_dir):
+    # Exclude directories from list of trusted files
+    exclude_dirs = ['boot', 'dev', 'etc/rc', 'proc', 'sys', 'var']
+    exclude_re = re.compile('^/(' + '|'.join(exclude_dirs) + ').*')
+    num_trusted = 0
+    trusted_files = ''
+    script_file = os.path.basename(__file__)
+
+    for root, _, files in os.walk(root_dir, followlinks=False):
```

Comment:
@vahldiek We have discussed this topic in the past. The current design of doing os.walk, excluding certain folders and taking the vast majority of files, can include system libraries (not needed by the original application), as trusted files, and some of them can have security vulnerabilities. Although this over-estimation design is ok for this initial version, we need to explore, if there is a better option for some future version of this tool.